suite-native/atoms/src/StepsProgressBar.tsx

```diff
@@ -29,10 +29,10 @@ const progressBarItemStyle = prepareNativeStyle<{ isActive: boolean }>((utils, {
     borderRadius: utils.borders.radii.small / 4,
     borderWidth: utils.borders.widths.medium,
     borderColor: isActive
-        ? utils.colors.backgroundPrimaryDefault
+        ? utils.colors.backgroundSecondaryDefault
         : utils.colors.backgroundNeutralSubdued,
     backgroundColor: isActive
-        ? utils.colors.backgroundPrimaryDefault
+        ? utils.colors.backgroundSecondaryDefault
         : utils.colors.backgroundNeutralSubdued,
```

Comment:
btw I think you don't use icons semantically according their designation. You should use `borderElevation0` etc.